build.sh

```diff
@@ -24,6 +24,7 @@ build planet-venus debian-10-slim latest
 build kubectl okd-c1 latest
 
 build hugo-node h0.62.2-n10.15.0
+build hugo-node h0.96.1-n16.15.0
```

Comment:
Hm, I guess, a image name "h0.96.1-n16.15.0" for version "v0.99.1" will cause irritation. Or is the a special consideration, I don't know?

Reply:
It's a typo! Thanks for pointing it out! I fixed it now!